SU-HSDO/suhumsci
# Four Kitchens Release - Sprint 36

# IN PROGRESS

## Summary
- SHS-5072: Add paragraph browser thumbnails for all private components
- SHS-5091: A11y Level A: Empty heading, on view on Bing Nursery
- SHS-4912: Implementation: Change method for making entire vertical linked cards clickable

## Need Review By (Date)
10/16

## Urgency
medium

## Steps to Test

SHS-5072

1. Visit a page like https://hs-traditional.suhumsci.loc/admin/structure/paragraphs_type and see the Private Collection and Private Text Area thumbnails
2. Edit a Private page like https://hs-traditional.suhumsci.loc/private-pages/private-page-components
3. Click Add Paragraph at the bottom of the page
4. Note the Private Text Area image and the Private Collection image on the Paragraph Browser modal that pops up. 

SHS-5091

1. Visit a page like https://bingschool.suhumsci.loc/news-events/bing-times
2. Ensure that there is no longer an HTML error on "2023 Issue" title (halfway down the page) where the template prints an open `<h3>` tag, self-closes, and then the `<h2>2023 Issue</h2>` prints after it. 

SHS-4912

1. Visit vertical linked cards in Traditional and Colorful themes:
https://hs-colorful.suhumsci.loc/components/collections-cards
https://hs-traditional.suhumsci.loc/components/collections-cards
2. Note that the entire card is clickable
3. Note that the link within the description field is clickable
4. Note that both links can be tabbed to and have a visible focus state
5. Note that the new theming does not produce any accessibility errors

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
